api/src/test/java/com/epam/pipeline/manager/docker/DockerRegistryNotificationTest.java

```diff
@@ -16,326 +16,389 @@
-        aclTestDao.createAclClassIfNotPresent(registryAclClass);
-
-        AclTestDao.AclObjectIdentity registryIdentity = new AclTestDao.AclObjectIdentity(testUserSid, registry.getId(),
-                                                                 registryAclClass.getId(), null, true);
-        aclTestDao.createObjectIdentity(registryIdentity);
-
-        AclTestDao.AclObjectIdentity registryIdentity2 = new AclTestDao.AclObjectIdentity(adminSid, registry2.getId(),
-                registryAclClass.getId(), null, true);
-        aclTestDao.createObjectIdentity(registryIdentity2);
-
-        AclTestDao.AclObjectIdentity registryIdentity3 = new AclTestDao.AclObjectIdentity(adminSid, registry3.getId(),
-                registryAclClass.getId(), null, true);
-        aclTestDao.createObjectIdentity(registryIdentity3);
-
-        AclTestDao.AclObjectIdentity groupIdentity = new AclTestDao.AclObjectIdentity(testUserSid, group.getId(),
-                                                                groupAclClass.getId(), null, true);
-        aclTestDao.createObjectIdentity(groupIdentity);
-
-        // All Test users can write to registry
-        AclTestDao.AclEntry registryAclEntry = new AclTestDao.AclEntry(registryIdentity, 1, adminSid,
-                                                                       AclPermission.WRITE.getMask(), true);
-        aclTestDao.createAclEntry(registryAclEntry);
-
-        registryAclEntry.setSid(testUserSid);
-        registryAclEntry.setOrder(2);
-        aclTestDao.createAclEntry(registryAclEntry);
-
-        // All Test users can write to group
-        AclTestDao.AclEntry groupAclEntry = new AclTestDao.AclEntry(groupIdentity, 1, adminSid,
-                AclPermission.WRITE.getMask(), true);
-        aclTestDao.createAclEntry(groupAclEntry);
-
-        groupAclEntry.setSid(testUserSid);
-        groupAclEntry.setOrder(2);
-        aclTestDao.createAclEntry(groupAclEntry);
-
-        //Only Admin can write to second registry
-        AclTestDao.AclEntry registryAclEntry2 = new AclTestDao.AclEntry(registryIdentity2, 1, adminSid,
-                AclPermission.WRITE.getMask(), true);
-        aclTestDao.createAclEntry(registryAclEntry2);
-
-        AclTestDao.AclEntry registryAclEntry3 = new AclTestDao.AclEntry(registryIdentity3, 1, adminSid,
-                AclPermission.WRITE.getMask(), true);
-        aclTestDao.createAclEntry(registryAclEntry3);
-        registryAclEntry3.setSid(testUserSid);
-        registryAclEntry3.setOrder(2);
-        aclTestDao.createAclEntry(registryAclEntry3);
+    @Autowired
+    private UserManager mockUserManager;
 
-        MockitoAnnotations.initMocks(this);
+    @InjectMocks
+    private final DockerRegistryManager registryManager = new DockerRegistryManager();
 
-        TestUtils.configureDockerClientMock(dockerClient, dockerClientFactoryMock);
+    private DockerRegistry registry1;
```

Comment:
Maybe we shall refactor this test and grant more accurate names. For example:
```
registry2 -> registryWithExternalPath
registry3 -> registryWithoutWriteAccess
```

Reply:
Done!